content/en/synthetics/browser_tests/_index.md

```diff
@@ -62,7 +62,7 @@ Define the configuration of your browser test.
 
 3. **Name**: The name of your browser test.
 4. **Select your tags**: The tags attached to your browser test. Use the `<KEY>:<VALUE>` format to filter on a `<VALUE>` for a given `<KEY>` on the Synthetic tests page.
-5. **Browsers & Devices**: The browsers (`Chrome`, `Firefox`) and devices (`Laptop Large`, `Tablet`, `Mobile Small`) to run your test on.
+5. **Browsers & Devices**: The browsers (`Chrome`, `Firefox`, `Edge`) and devices (`Laptop Large`, `Tablet`, `Mobile Small`) to run your test on. **Note:** 
```

Comment:
Did you mean to add the **Note:** here?

Reply:
Oops, thanks Kari - getting rid of this one!